### PR TITLE
Feature/dswp 55 horizontal card pattern

### PR DESCRIPTION
**Pull request by ASpiteri-BCGov**

### CODE REVIEW

File: src/styles/patterns/dswp-large-image-text.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+	gap: 0;
+	border-radius: 8px;
+	border: 1px solid var(--dswp-surface-color-border-default) !important;
+
+	img {
+		height: 222px;
+		border-radius: 8px 0 0 8px;
+		padding: 1rem;
+		background: #faf9f8;
```
Comment (mhaswell-bcgov) [inline src/styles/patterns/dswp-large-image-text.scss:42]: Is there a design system token for this color?

File: src/styles/patterns/dswp-large-image-text.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	.dswp-large-image-text {
+
+		img {
+			width: 100% !important;
+		}
+	}
+
+}
+
+@media (min-width: $break-mobile) and (max-width: $break-medium) {
```
Comment (mhaswell-bcgov) [inline src/styles/patterns/dswp-large-image-text.scss:12]: These could be refactored into a nested structure to remove a little bit of repetition.

File: functions.php [hunk 3]
```diff
@@ -52,3 +52,16 @@ function design_system_include_block_style_variations( $dir_path ) {
...
 $dir_path = get_template_directory() . '/blocks/core/style-variations';
 design_system_include_block_style_variations( $dir_path );
+
+
+/**
+ * Restrict access to the locking UI to Administrators.
+ *
+ * @param array $settings Default editor settings.
+ */
+function example_theme_restrict_locking_ui( $settings ) {
```
Comment (mhaswell-bcgov) [inline functions.php:62]: Is this a feature we're adding or just trying something out? I think if it's a new feature it should be in the changelog and be given a better name.

File: patterns/dswp-horizonal-card.php [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+<!-- wp:column {"verticalAlignment":"stretch","width":"74%","style":{"spacing":{"padding":{"top":"0","bottom":"0","right":"var:preset|spacing|40","left":"var:preset|spacing|40"}}},"layout":{"type":"default"}} -->
+<div class="wp-block-column is-vertically-aligned-stretch" style="padding-top:0;padding-right:var(--wp--preset--spacing--40);padding-bottom:0;padding-left:var(--wp--preset--spacing--40);flex-basis:74%"><!-- wp:spacer {"height":"1rem"} -->
+<div style="height:1rem" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:heading {"level":5,"style":{"spacing":{"margin":{"top":"0","bottom":"0"}}}} -->
+<h5 class="wp-block-heading" style="margin-top:0;margin-bottom:0">Neque porro quisquam</h5>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"style":{"typography":{"fontSize":"16px"}}} -->
```
Comment (mhaswell-bcgov) [inline patterns/dswp-horizonal-card.php:28]: Font size is hardcoded here. Does this need to always be exactly 16px or can it use a font size from the theme?

**Review by mhaswell-bcgov — CHANGES_REQUESTED** [review]

**Comment by ASpiteri-BCGov** [comment]
@mhaswell-bcgov resolved all issues.

**Review by mhaswell-bcgov — APPROVED** [review]

**Pull request merged**